{
  "channeldata": {
    "activate.d": false,
    "binary_prefix": false,
    "deactivate.d": false,
    "description": null,
    "dev_url": "https://github.com/dougthor42/PyErf",
    "doc_source_url": null,
    "doc_url": "https://github.com/dougthor42/PyErf",
    "home": "https://www.github.com/dougthor42/pyerf",
    "icon_hash": null,
    "icon_url": null,
    "identifiers": null,
    "keywords": null,
    "license": "GPL-3.0",
    "post_link": false,
    "pre_link": false,
    "pre_unlink": false,
    "recipe_origin": null,
    "run_exports": {},
    "source_git_url": null,
    "source_url": "https://pypi.io/packages/source/p/pyerf/pyerf-1.0.1.tar.gz",
    "subdirs": [
      "noarch"
    ],
    "summary": "A pure-Python implementation of the error function and inverse error function.",
    "tags": null,
    "text_prefix": false,
    "timestamp": 1582399695,
    "version": "1.0.1"
  },
  "channeldata_version": 1,
  "feedstock": null,
  "labels": [
    "main",
    "cf202003"
  ],
  "package": "pyerf-1.0.1-py_0.tar.bz2",
  "repodata": {
    "build": "py_0",
    "build_number": 0,
    "depends": [
      "python"
    ],
    "license": "GPL-3.0",
    "license_family": "GPL",
    "md5": "184720be08034504c5c2f3f3402cc1d7",
    "name": "pyerf",
    "noarch": "python",
    "sha256": "26cb728da82cc4f406394566654d5f536a4f9aa43bb13080b971292c0e3b96da",
    "size": 25524,
    "subdir": "noarch",
    "timestamp": 1582399695657,
    "version": "1.0.1"
  },
  "repodata_version": 1,
  "subdir": "noarch",
  "url": "https://conda.anaconda.org/conda-forge/noarch/pyerf-1.0.1-py_0.tar.bz2"
}